qa/tasks/cephfs/test_killpoints_dir_export.py

```diff
@@ -0,0 +1,136 @@
+
+"""
+Test to exercise killpoints available during directory export code path
+After hitting a killpoint, MDS crashes and cluster must find a standby MDS
+to become active. Exported data must be available after mds replacement.
+
+"""
+import logging
+from tasks.cephfs.cephfs_test_case import CephFSTestCase
+
+log = logging.getLogger(__name__)
+
+
+class TestKillPoints(CephFSTestCase):
+    # Two active MDS and a standby
+    MDSS_REQUIRED = 3
+
+    init = False
+
+    def setup_cluster(self):
+        # Set Multi-MDS cluster
+        self.fs.set_allow_multimds()
```

Comment:
The test immediately fails here because there is no such function on the Filesystem class.  Possibly you had a change that didn't get committed here?

If you are adding a helper for this, then I'd suggest making a bit more general, something like "set_flag" and take the "allow_multimds" part as an argument so that we don't end up with a separate function for each flag.